recipes/libtommath/all/conanfile.py

```diff
@@ -21,119 +28,72 @@ class LibTomMathConan(ConanFile):
-
-    def _run_makefile(self, target=None):
-        target = target or ""
-        autotools = AutoToolsBuildEnvironment(self)
-        autotools.libs = []
-        if self.settings.os == "Windows" and self.settings.compiler != "Visual Studio":
-            autotools.link_flags.append("-lcrypt32")
-        if self.settings.os == "Macos" and self.settings.arch == "armv8":
-            # FIXME: should be handled by helper
-            autotools.link_flags.append("-arch arm64")
-        args = autotools.vars
-        args.update({
-            "PREFIX": self.package_folder,
-        })
-        if self.settings.compiler != "Visual Studio":
-            if tools.get_env("CC"):
-                args["CC"] = tools.get_env("CC")
-            if tools.get_env("LD"):
-                args["LD"] = tools.get_env("LD")
-            if tools.get_env("AR"):
-                args["AR"] = tools.get_env("AR")
-
-            args["LIBTOOL"] = "libtool"
-        arg_str = " ".join("{}=\"{}\"".format(k, v) for k, v in args.items())
-
-        with tools.environment_append(args):
-            with tools.chdir(self._source_subfolder):
-                if self.settings.compiler == "Visual Studio":
-                    if self.options.shared:
-                        target = "tommath.dll"
-                    else:
-                        target = "tommath.lib"
-                    with tools.vcvars(self):
-                        self.run("nmake -f makefile.msvc {} {}".format(
-                            target,
-                            arg_str,
-                        ), run_environment=True)
-                else:
-                    if self.settings.os == "Windows":
-                        makefile = "makefile.mingw"
-                        if self.options.shared:
-                            target = "libtommath.dll"
-                        else:
-                            target = "libtommath.a"
-                    else:
-                        if self.options.shared:
-                            makefile = "makefile.shared"
-                        else:
-                            makefile = "makefile.unix"
-                    self.run("{} -f {} {} {} -j{}".format(
-                        tools.get_env("CONAN_MAKE_PROGRAM", "make"),
-                        makefile,
-                        target,
-                        arg_str,
-                        tools.cpu_count(),
-                    ), run_environment=True)
+        if Version(self.version) >= "1.3":
+            get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        else:
+            get(self, **self.conan_data["sources"][self.version]["source"], strip_root=True)
+            download(self, **self.conan_data["sources"][self.version]["cmakelists"], filename="CMakeLists.txt")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
```

Comment:
As per the new policies we're trying to follow, if upstream does not support windows shared builds we should not rely on exporting all symbols by default